scc/cert/aggregated_signature_json.go

```diff
@@ -0,0 +1,46 @@
+package cert
+
+import (
+	"fmt"
+
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/bls"
+	"github.com/0xsoniclabs/sonic/utils/jsonhex"
+)
+
+// AggregatedSignatureJson is a JSON friendly representation of an AggregatedSignature.
+type AggregatedSignatureJson[S Statement] struct {
+	// signers:  BitSet[scc.MemberId]
+	Signers jsonhex.Bytes `json:"signers" gencodec:"required"`
```

Comment:
You have implemented JSON marshaling for the `BitSet[T]` but you ignore it here?